aircraft/cost.py

```diff
@@ -0,0 +1,105 @@
+"""Aircraft cost models"""
+from gpkit.shortcuts import Var, Model
+import gpkit
+gpkit.disable_units()
+
+class DAPCA4Cost(Model):
+    """
+    Cost Model from Raymer, Aircraft Design: A Conceptual Approach 3rd Edition,
+    pp 713-714
+
+    Note: this is also one instance of the RAND Corp cost models.
+
+    Cost is in 2012 dollars
+
+    Units disabled because '$' is not defined as a unit 
+
+    Labor Rates are found from Raymer
+
+    Default values based on Honda Jet specs
+
+    Arguments
+    ---------
+    engine_defined : bool
+        If true, must specify T_max, M_max, and T_tur
+        If false, must define engine_cost
+        Todo: this should really be replaced with an engine cost (sub) model.
+    """
+    def setup(self, W_e=9200, V=420, Q=5, FTA=1, N_eng=10, T_max=2050,
+              M_max=0.63, T_tur=2050, R_avn=5000, engine_cost=500000,
+              eng_defined=False):
```

Comment:
@mjburton11, note our new suggested best practice is to put these default values in the `Variable` constructor, instead of in the kwargs of `setup`.